app/models/move.rb

```diff
@@ -73,7 +77,11 @@ def from_nomis?
   end
 
   def existing
-    Move.find_by(date: date, person_id: person_id, from_location_id: from_location_id, to_location_id: to_location_id)
+    self.class.find_by(date: date, person_id: person_id, from_location_id: from_location_id, to_location_id: to_location_id)
```

Comment:
`find_by` only ever returns one record (although the order is not guaranteed). See https://api.rubyonrails.org/classes/ActiveRecord/FinderMethods.html#method-i-find_by

Hence I don't think it's particularly valuable to add a test for the scenario of multiple existing moves

Reply:
should `status` also be a factor in the `find_by` call? otherwise you may find legitimate cancelled moves, rather than the dodgy duplicated move?

Reply:
GPWM!